pyccel/codegen/printing/cwrappercode.py

```diff
@@ -292,7 +292,7 @@ def static_function_signature(self, expr):
             Signature of the function.
         """
         #if target_language is C no need for the binding
-        if self._target_language == 'c':
+        if self._target_language in ('c', 'ccuda'):
```

Comment:
@EmilyBourne this might need to change later if we want to bind `Ccuda` types

Reply:
That said the wrapper should be changed soon anyway so maybe this is ok for now

Reply:
> minus some specific types and function decorators.

Function decorators in cuda code?
If the types appear in the function declarations then we will indeed need a wrapper.

I imagine it also depends on how we plan to manage functions on the device. Should these be called by other translated functions or should the wrapper transfer the data to GPU to call the function?

Reply:
by a decorator i mean `__global__` declaration specifier.
and there is some types that are `Ccuda` specifics like complex.
for now i am always wrapping `kernel` functions inside another simple `C` function because we do not have a proper wrapper.
I do not think there will be any problem with the function on the `device` because all of them can only interact with the `kernel`.